ckanext/civity/harvesters/civity_harvester.py

```diff
@@ -544,3 +564,32 @@ def _get_package_name(self, harvest_object, title):
             name = package.name
 
         return name
+
+    def resource_create_default_resource_views(self, context, resource_dict):
+        result = True
+        data_dict = {
+            'resource': resource_dict,
+            'create_datastore_views': False
+        }
+
+        try:
```

Comment:
Make a wrapper for calling these actions, because it's always the same:
action name, data_dict and the handling of the errors